packages/monitor-v2/src/monitor-polymarket/common.ts

```diff
@@ -134,55 +137,76 @@ export const formatPriceEvents = async (
         ancillaryData: event.args.ancillaryData,
         proposedPrice: ethers.utils.formatEther(event.args.proposedPrice),
         eventIndex: event.logIndex,
+        oracleAddress: event.address,
       };
     })
   );
 };
 
 export const getPolymarketMarkets = async (params: MonitoringParams): Promise<PolymarketMarket[]> => {
+  const markets = [];
+  const pagination = 100;
+  let offset = 0;
```

Comment:
How were we getting away with no pagination before? Was it just giving us the first 100 or so?